### ISSUE TITLE
Paths to long for extraction after download - can't use without composer

### ISSUE THREAD

**Issue by flyinRyan00** [googleapis/google-api-php-client#1857]
Some issues with installation: composer is not an option as I don't have access to it (I'm on a shared server), so I go to download, unzip and upload to the server. However, paths are too long to unzip on Windows (there is a 260 character limit). This effectively means anyone without composer who runs windows can't use it.

**Comment by oleg972** [comment]
cute:http 404 request

пн, 22 июн. 2020 г. в 21:59, flyinRyan00 <notifications@github.com>:

**Comment by bshaffer** [comment]
Hi @flyinRyan00 

I took a look through the files in the zipped package, and the longest filename is 100 characters and the longest filepath was 177 characters

```
Longest File: GoogleCloudDialogflowV2beta1IntentMessageBrowseCarouselCardBrowseCarouselCardItemOpenUrlAction.php
Longest FilePath: ./vendor/google/apiclient-services/src/Google/Service/ManagedServiceforMicrosoftActiveDirectoryConsumerAPI/GoogleCloudSaasacceleratorManagementProvidersV1ProvisionedResource.php
```

Yes this is very long, but it shouldn't break your 260-character limit.

**Comment by bshaffer** [comment]
@flyinRyan00 can you provide more information on how to reproduce this error?

**Comment by danoscarmike** [comment]
Touching in on this one - @flyinRyan00 are you still blocked on this? Can you help us to reproduce if so?

**Comment by bshaffer** [comment]
Closing due to inactivity

**Comment by QuietWindUponTheMoor** [comment]
This issue can be fixed by changing "LongPathsEnabled" to true/1 in regedit, however I want to add that professional devs shouldn't be making file names that are 118+ characters long. That is absolutely absurd and unnecessary. Yes, I counted. Multiple of the files in the PHP client for PHP8.0 are 100+ characters long for a SINGLE FILE NAME. Not just the entire path, but the FILE ITSELF. I saw another issue that called out the same thing, and the OP's comments were removed.
